docs/config/node.rst

```diff
@@ -120,6 +120,20 @@ General
    effect for new sessions.
 
 
+.. _statement_max_length:
+
+**statement_max_length**
+  | *Default:* ``65536``
+  | *Runtime:* ``no``
```

Comment:
it's `yes` here, right?

Reply:
No it currently can't be changed at runtime.

Reply:
I was already unsure if a setting is warranted at all. Still tempted to remove it altogether.

Reply:
What's the catch, why are you hesitant with the setting?

Reply:
Makes things more complex. More that people will read in the documentation and wonder if they should change it. And if set too large and used it will result in OOMs.

On the other hand, having it it makes the test case a bit faster.

Reply:
65536 (64KB) is on the lower end for such a setting though. 
Postgres limits to 1GiB
MySQL, Snowflake, SQL Server and others to at least 1 MB

Reply:
Not arguing for the limit value itself, we should set something which is sensible and doesn't affect the common use cases. Are you against the setting as a whole though @proddata ?

Reply:
No objections to having such a setting, if it’s deemed applicable to limit the issue.
Changing the setting should be a rare exception though and should not cause issues for users of third-party software that generates SQL statements automatically, such as BI tools like Tableau or ORMs.

Reply:
The biggest documented statements I can quickly find are in the area of ~ 100 KB. i.e. a limit of 128 or 256 might cut it.